python/setup.py

```diff
@@ -177,10 +63,15 @@ def build_extensions(self):
     description='python interface to FINUFFT',
     long_description='python interface to FINUFFT (Flatiron Institute Nonuniform Fast Fourier Transform) library.',
     license="Apache 2",
-    ext_modules=ext_modules,
     packages=['finufft'],
-    install_requires=['numpy','pybind11>=2.2','python-dotenv'],
-    cmdclass={'build_ext': BuildExt},
-    zip_safe=False
+    install_requires=['numpy','python-dotenv'],
+    zip_safe=False,
+    py_modules=['finufftc'],
```

Comment:
What does this do?

Reply:
This tells python to expect a module by that name.  The lines below create the matching extension module.